### DISCUSSION TITLE
Select + Combobox but without aria-select=true to the highlighted item

### DISCUSSION THREAD

**Discussion by cloud-walker** [ariakit/ariakit#2405] [Questions]
Hi!  I want to achieve a dropdown similar to the [Select with Combobox](https://ariakit.org/examples/select-combobox) example, but:

1. I want the aria-selected attribute to be actually true only when the item is selected, like in the [Multi-selectable Combobox](https://ariakit.org/examples/combobox-multiple) example
2. Sometimes I need it to be multi-selectable

Any advice? Its something standard in terms of accessibility?

**Comment by cloud-walker** [comment]
Maybe I found a solution: https://codesandbox.io/p/sandbox/magical-rain-hvhggu

**Reply by diegohaz** [reply]
I'd add ` composite={false}` to `SelectPopover`, otherwise it will conflict with `ComboboxList` (both are composite widgets).

**Reply by cloud-walker** [reply]
So I'm in the right direction? If so, I'm wondering why the select + combobox examples are not all setup this way, I mean, when composing a select with a combobox you always want the select behaviour on the items, no?
